test/spec.yml

```diff
@@ -720,6 +721,26 @@
       out: |-
           "Hello World!"
 
+- test: Test if imagemagick exists in bolt
+  setup:
+      - bn create node8 imagemagick
+      - npm i shelljs
+      - |-
+        cat <<EOF >/home/dockeruser/test/function.js
+        
+        const shell = require('shelljs');
+        exports.handler = async() => {
+          return shell.exec('convert --version');
```

Comment:
Additional npm dependency can be avoided by using `child_process.execSync('convert --version', { econding: 'utf8' });`

Reply:
I considered it at the time but I consider `shelljs` to be cleaner. I'm of the belief that most users wouldn't use `execSync` they would favor the npm package.

Reply:
The top stackoverflow comment does not mention `shelljs`, but some do - it seems like an overkill since `shelljs` is "a portable (Windows/Linux/OS X) implementation of Unix shell commands on top of the Node.js API".

However, this comment is specifically about an npm dependency to tests that are somewhat flaky.

Reply:
You may be right regarding usage (not sure). 

I definitely don't understand this comment

> However, this comment is specifically about an npm dependency to tests that are somewhat flaky.

The binaris.yml spec tests are only as flaky as the underlying system.

Reply:
iiuc, the npm dependency adds flakiness in this case.

Reply:
There's flakiness of the underlying system (which we try to stress by running parallel tests), which is interesting (signal).

This adds the flakiness of `npm i` which is not related to the underlying system (noise).